string_lab.js

```diff
@@ -0,0 +1,197 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+//
+//
+let str = 'Hello Stephanie'
+let firstHalf = str.slice(0,5)
+let secondHalf = str.slice(6, str.length)
+console.log(firstHalf)
+
+//
+// 2. `console.log` the length of a string.
+let name = 'Stephanie'
+console.log(name.length)
+
+
+// // 3. `console.log` the last character of any string.
+let steph= 'Good morning Stephanie'
```

Comment:
Same problem here as above. What would happen if I change steph to "Buenos Dias Stephanie"? This is called hard coding which is a bad practice. Your code must be able to accommodate to different inputs and returns consistent and correct outputs.